### PR TITLE
[bugfix] Hide speaker dropdown for safari browser

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What

In configuration page, hide speaker dropdown for safari browser
image

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/3398753
image

# How Tested

Locally on MacOs Safari browser. Call Composite.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

**Comment by JamesBurnside** [comment]
Hmm I suppose we should remove this empty space for this scenario
image

**Comment by edwardlee-msft** [comment]
> Hmm I suppose we should remove this empty space for this scenario -- might need @pereiralex input image

Confirmed with Alex P that we should shrink the container size. 
New commit addresses those issues. Please refer to the updated screenshot provided in the description of this pr.

File: packages/react-composites/src/composites/CallComposite/styles/CallConfiguration.styles.ts [hunk 1]
```diff
@@ -91,10 +91,10 @@ export const configurationSectionStyle: IStackStyles = {
 /**
  * @private
  */
-export const selectionContainerStyle = (theme: ITheme): string =>
+export const selectionContainerStyle = (theme: ITheme, browser?: string): string =>
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/styles/CallConfiguration.styles.ts:94]: nit: difficult to understand why safari means the height is different, suggestion: use something like `noSpeakerDropdownShown` as the variable name here

File: packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx [hunk 2]
```diff
@@ -117,6 +117,8 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
 
   const cameraPermissionGranted = props.cameraPermissionGranted;
   const micPermissionGranted = props.microphonePermissionGranted;
+  const isSafariBrowser = adapter.getState().environmentInfo?.environment.browser.toLowerCase() === 'safari';
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx:120]: Could this instead be isSafariBrowser **&& speakers.length === 0**? My worry is one day safari will suddenly expose changing speakers independently

**Review by JamesBurnside — APPROVED** [review]
Couple suggestions - looks good though!

**Review by JamesBurnside — APPROVED** [review]

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**